types/altair/light_client.yaml

```diff
@@ -69,3 +69,18 @@ Altair:
         $ref: './sync_aggregate.yaml#/Altair/SyncAggregate'
       signature_slot:
         $ref: '../primitive.yaml#/Uint64'
+  LightClientInstantUpdate:
+    type: object
+    properties:
```

Comment:
should probably add the required attrbiutes list here if we're not requiring all fields - sounds like `new_head_header` is optional...

Also is `new_head_header` in a spec somewhere?  I'm not a huge fan of that name...

Reply:
Renamed to `new_head`, the description and type reference should tell the rest. Also added the required attrbiutes list.